### DISCUSSION TITLE
If a postgresql is on the same localhost, allow by default to avoid Error: Migration engine error: db error: ERROR: permission denied for database databasename

### DISCUSSION THREAD

**Discussion by miguelgargallo** [prisma/prisma#15763] [Q&A]
If a postgresql is on the same localhost, allow by default to avoid Error: Migration engine error: db error: ERROR: permission denied for database databasename

**Comment by nurul3101** [comment]
Hi @miguelgargallo 👋 
Can you please elaborate? Did you mean to create a Feature Request?

**Reply by vladi-stevanovic** [reply]
👋 Hello @miguelgargallo did you perhaps have a chance to check Nurul's questions? 
Let us know if this is still an open issue or if you've already solved it, we'd love to know how! 

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
